### ISSUE TITLE
Cannot open a direct link

### ISSUE THREAD

**Issue by wocmultimedia** [qTranslate-Team/qtranslate-x#360]
Hi,
I'm using the plugin setting Pre-Path but I cannot open a direct link like this http://www.wocmultimedia.biz/buddha/FLIPKARMA/index.html  from Menu Custom Link because the plugin automatically add the pre-path language in this way
http://www.wocmultimedia.biz/it/buddha/FLIPKARMA/index.html
resulting in a 404 error page not found
How can I solve it?
Thanks
Angelo

**Comment by johnclause** [comment]
Seems like the issue is resolved? URL http://www.wocmultimedia.biz/it/buddha/FLIPKARMA/index.html works now?

I am closing it for now, we can still write into closed issue or we may reopen it if needed.

**Comment by wocmultimedia** [comment]
Hi John,
I'm sorry but I cannot open it, still gives error 404. Please, make a test yourself. I'm using Chrome and Firefox to test it. I set a link in the menu you can see it is labelled test link on main menu. I just add the link http://www.wocmultimedia.biz/buddha/FLIPKARMA/index.html but try to click and see still gives the error. You have the Admin data to access. Thanks. Angelo

**Comment by johnclause** [comment]
Indeed, neutral URL http://www.wocmultimedia.biz/buddha/FLIPKARMA/index.html works, but language encoded versions, like http://www.wocmultimedia.biz/it/buddha/FLIPKARMA/index.html or http://www.wocmultimedia.biz/en/buddha/FLIPKARMA/index.html do not work. I think it is because of the extension .html, which is considered to be static page and redirection rules are not applied. So, this url should not be converted to language encoded one in the menu, but it is. This is indeed a coding bug, which needs to be fixed.

As a workaround for now try using "setlang=no" in the menu configuration, as eaplained here:
https://qtranslatexteam.wordpress.com/faq/#CustomMenuConvertURL

**Comment by wocmultimedia** [comment]
Hi John,
Thanks for the precious support!
As you stated the link http://www.wocmultimedia.biz/buddha/FLIPKARMA/index.html is automatically converted adding the pre-path but it shouldn't be. Anyhow, I followed your suggestion and adding 
"?setlang=no" fixed the issue for the moment. It's a good workaround.
Thanks.
Have a good time.
Angelo
